tools/debugging/stress_test_transfers.py

```diff
@@ -510,37 +510,42 @@ def main() -> None:
     nodes_config: List[NodeConfig] = list()
     nodes_running: List[RunningNode] = list()
 
+    defaults = {
+        "--log-config": "raiden:DEBUG",
+        "--environment-type": "development",
+        "--datadir": datadir,
+    }
+
     for section in config:
         if NODE_SECTION_RE.match(section):
             node_config = config[section]
             address = node_config["address"]
             port = next(port_generator)
             api_url = f"{interface}:{port}"
 
+            node = defaults.copy()
+            node.update(
+                {
+                    "--keystore-path": node_config["keystore"],
+                    "--password-file": node_config["password-file"],
+                    "--eth-rpc-endpoint": node_config["ethnode"],
+                    "--network-id": node_config["networkid"],
```

Comment:
We could use the same names in the config file that raiden uses. That would be more consistent and we would be able to pass through all options without explicitly adding support for new options.

Reply:
done.